### PR TITLE
Wake up from sleep

### PR DESCRIPTION
**Pull request by alexbn71**
This function just call autoBaud() to wake up the module from sleep (As mentioned in the RN2483 datasheet 2.3.1).

### CODE REVIEW

File: src/TheThingsNetwork.h [hunk 1]
```diff
@@ -93,6 +93,7 @@ class TheThingsNetwork
   ttn_response_t poll(port_t port = 1, bool confirm = false);
   void sleep(uint32_t mseconds);
   void saveState();
+  void wake();
```
Comment (johanstokking) [inline src/TheThingsNetwork.h]: Move one line up, after `sleep()`, because these belong together

File: src/TheThingsNetwork.cpp [hunk 2]
```diff
@@ -958,3 +958,8 @@ void TheThingsNetwork::sleep(uint32_t mseconds)
   modemStream->write(SEND_MSG);
   debugPrintLn(buffer);
 }
+
+void TheThingsNetwork::wake()
+{ 
+	autoBaud();
```
Comment (johanstokking) [inline src/TheThingsNetwork.cpp]: Spacing, add one space for consistent formatting

**Comment by johanstokking** [comment]
@alexbn71 good to go. Can you add yourself to the `AUTHORS` file too?

**Comment by alexbn71** [comment]
with great pleasure... Can I add AUTHORS now to the current PR? or new one PR?

**Comment by johanstokking** [comment]
Released in https://github.com/TheThingsNetwork/arduino-device-lib/releases/tag/v2.5.5

**Pull request merged**